sw/host/tests/chip/sysrst_ctrl/sysrst_ctrl.rs

```diff
@@ -50,3 +50,17 @@ pub fn set_pins(transport: &TransportWrapper, config: &Config, values: u8) -> Re
     }
     Ok(())
 }
+
+/// Read pins: the i-th bit of `values` corresponds to the i-th entry of `config.input_pins`.
+#[allow(dead_code)]
+pub fn read_pins(transport: &TransportWrapper, config: &Config) -> Result<u8> {
+    let mut values = 0u8;
+    // Set pins on the transport.
+    for (i, pin) in config.input_pins.iter().enumerate() {
```

Comment:
Should we add something like `ensure!(config.input_pins.len() <= 8);` just to avoid silent bugs where we overflow the `u8`?